cc.nift.nsm.json

```
@@ -0,0 +1,23 @@
+{
+    "app-id": "cc.nift.nsm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "nsm",
+    "modules": [
+        {
+            "name": "nsm",
+            "buildsystem": "simple",
+            "build-commands": [
+				"make",
+				"make install"
+            ],
+            "sources": [
+                {
+					"type": "git",
+					"url": "https://github.com/nifty-site-manager/nsm-flatpak"
```

You should define a commit/tag to build from.

```suggestion
					"url": "https://github.com/nifty-site-manager/nsm-flatpak.git",
					"tag": "v1.0"
```

---

A tag is not enough as you can still push to it. A tag with a commit or just commit is what we need for Flathub as we want to be able builds to be reproducible.